shapeshift/web
# feat: savers withdraw not enough funds

## Description

QoL for ops to stop receiving "WHERE ME MONESE" tickets.

## Pull Request Type


## Issue (if applicable)



closes https://github.com/shapeshift/web/issues/5821

## Risk



None?

## Testing



- Have a UTXO savers position, but not enough fundus for withdraw + reconciliation (e.g BTC, which when you take the two into account, can be quite a high missing amount)
- Ensure the copy looks good
- Ensure the displayed address looks good
- Ensure the copy button copies your address to send fundus to

### Engineering



- ☝🏽 

### Operations



- ☝🏽 

## Screenshots (if applicable)

<img width="549" alt="Screenshot 2023-12-18 at 21 22 27" src="https://github.com/shapeshift/web/assets/17035424/d04d853d-4c25-4954-8532-0fa83af2645f">
